daemon/exec.go

```diff
@@ -306,6 +307,25 @@ func (daemon *Daemon) ContainerExecStart(ctx context.Context, name string, stdin
 	return nil
 }
 
+// ContainerExecKill kills an exec instance.
+func (daemon *Daemon) ContainerExecKill(ctx context.Context, name string, sig uint64) error {
```

Comment:
I think it would be best to move signal parsing here even if that means it is no longer consistent with container kill.